server/models/Review.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+
+const reviewSchema = new mongoose.Schema(
+  {
+    reviewerProfileId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Profile",
+    },
+    reviewedProfileId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Profile",
+    },
+    requestId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Request",
+    },
```

Comment:
Shouldn't any of these be `required`?

Reply:
Yes, I will add these in the following commit!